util/make_dir/make.ubuntu

```diff
@@ -13,7 +13,7 @@ CHAN            = MPI1
 COUPLE          = $(OASIS_HOME)
 #
 # ARCHDIR       : directory created when compiling
-ARCHDIR         = $(COUPLE)/Linux
+ARCHDIR         = $(COUPLE)/compile_oa3-mct
```

Comment:
Upstream they name this `compile_oa3-mct`? Not `compile_oasis3-mct`?

Reply:
Yes. https://gitlab.com/cerfacs/oasis3-mct/-/blob/OASIS3-MCT_5.0/util/make_dir/header_examples/make.ubuntu